RetrospectiveExtension.Frontend/components/feedbackCarousel.tsx

```diff
@@ -68,10 +75,38 @@ class FeedbackCarousel extends React.Component<IFeedbackCarouselProps, IFeedback
         {this.props.feedbackColumnPropsList.map((columnProps) => {
           const mainCardCount = columnProps.columnItems.filter((columnItem) => !columnItem.feedbackItem.parentFeedbackItemId).length;
 
+          columnProps.columnItems.forEach(columnItem => {
+            const feedbackItemProps = FeedbackColumn.createFeedbackItemProps(columnProps, columnItem, true);
+
+            // Establish whether an item in the column has children feedback grouped beneath it,
+            // and therefore will be the parent
+            const isGroupedCarouselItem = columnItem.feedbackItem.childFeedbackItemIds ? (columnItem.feedbackItem.childFeedbackItemIds.length > 0 ? true : false) : false;
```

Comment:
🤔 I wonder if the conditional can be simplified to 

`
columnItem.feedbackItem.childFeedbackItemIds ? (columnItem.feedbackItem.childFeedbackItemIds.length > 0) : false
`

Reply:
same on line 86

Reply:
Thank you for catching that. Ternary can be a tricky one!